superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -188,7 +188,7 @@ class Header extends React.PureComponent {
 
   forceRefresh() {
     if (!this.props.isLoading) {
-      const chartList = Object.values(this.props.charts);
+      const chartList = Object.keys(this.props.charts);
```

Comment:
I noticed this because when dashboard periodically refreshing for a long time, the dashboard state/chart state might be changed, like add filter etc. If I pass chart object here, its state is not updated. So i prefer to only pass the affected chart id.